src/config.ts

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+import * as dotenv from 'dotenv';
+import path from 'path';
+import type { AdditionalConfig, ProcessorConfig } from '@tazama-lf/frms-coe-lib/lib/config/processor.config';
+
+dotenv.config({
+  path: path.resolve(__dirname, '../.env'),
+});
+
+export interface ExtendedConfig {
+  DESTINATION_TRANSPORT_URL: string;
+  PRODUCER_STREAM: string;
+  KAFKA_TLS_CA?: string;
+  CLIENT_ID: string;
+  maxInFlightRequests: string;
+}
+
+export const additionalEnvironmentVariables: AdditionalConfig[] = [
+  {
+    name: 'DESTINATION_TRANSPORT_URL',
+    type: 'string',
+  },
+  {
+    name: 'PRODUCER_STREAM',
+    type: 'string',
+  },
+  {
+    name: 'KAFKA_TLS_CA',
+    type: 'string',
+    optional: true,
+  },
+  {
+    name: 'CLIENT_ID',
+    type: 'string',
+  },
+  {
+    name: 'maxInFlightRequests',
+    type: 'string',
+  },
+];
```

Comment:
Same as interface: 

No overlap with existing env (NATS: PRODUCER_STREAM)
Consistent casing
KAFKA prefix for grouping
Number for numeric env
```suggestion
export const additionalEnvironmentVariables: AdditionalConfig[] = [
  {
    name: 'KAFKA_DESTINATION_TRANSPORT_URL',
    type: 'string',
  },
  {
    name: 'KAFKA_PRODUCER_STREAM',
    type: 'string',
  },
  {
    name: 'KAFKA_TLS_CA',
    type: 'string',
    optional: true,
  },
  {
    name: 'KAFKA_CLIENT_ID',
    type: 'string',
  },
  {
    name: 'KAFKA_MAX_IN_FLIGHT_REQUESTS',
    type: 'number',
  },
];
```